Code/PgSQL/rdkit/README.md

```diff
@@ -116,6 +154,60 @@ The RDKit cartridge is now to ready to be used on the `nci5K` database.
 
 # Windows
 
+## Cartridge installation into conda PostgreSQL
+If you build the PostgreSQL cartridge against the conda `postgresql` package,
+CMake will find it without having to specify its location through CMake
+switches. Also installation will be straightforward as it will not require
+root privileges.
+Build RDKit in the usual way: configure with `cmake`, then build and install:
+```
+>cmake --build . --parallel %NUMBER_OF_PROCESSORS% --target install --config Release
+```
+You should see the following message:
+```
+=====================================================================
+PostgreSQL cartridge installation succeeded.
+=====================================================================
+Restart the PostgreSQL service before using the PostgreSQL
+RDKit cartridge.
+=====================================================================
+```
+To test the cartridge, first I initialize a database directory:
+```
+>initdb M:\postgresdb
+```
+I edit the M:\postgresdb\postgresql.conf configuration file
+to set the connection port to 6433 since on my system the default
+`5432` is already in use, then I start the PostgreSQL server:
+```
+>pg_ctl -D M:\postgresdb -l M:\postgresdb\logfile start
+waiting for server to start.... done
+server started
+```
+To test the RDKit cartridge, I create a a small test database:
+```
+>createdb -p 6433 nci5K
+```
+And attempt to load the cartridge in the database:
+```
+>psql -c 'create extension rdkit' -p 6433 nci5k
+CREATE EXTENSION
+```
+Should you see an error mesage such as
```

Comment:
small typo (message).

Reply:
@rvianello Thanks for spotting that! Fixed.